android/src/ti/map/TiUIMapView.java

```diff
@@ -865,6 +865,10 @@ public void onMapClick(LatLng point) {
 		}
 		clickablePolylines.clear();
 
+		KrollDict d = new KrollDict();
+		d.put(TiC.PROPERTY_LATITUDE, point.latitude);
+		d.put(TiC.PROPERTY_LONGITUDE, point.longitude);
+		proxy.fireEvent("mapclick", d);
```

Comment:
iOS also has the current instance / self inside the `map` key. For 100 % parity, that would be nice. Also nice would be to check if the event listener exists prior to firing the event. Not sure if Titanium Android can do that though.